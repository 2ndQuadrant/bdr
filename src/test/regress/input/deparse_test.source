---
--- DEPARSE_TEST
---

-- create roles used throughout the tests
create role clstr_user;
create role "current_user";
create role foreign_data_user;
create role "Public";
create role regressgroup1;
create role regressgroup2;
create role regression_bob;
create role regression_group;
create role regression_user1;
create role regression_user2;
create role regression_user3;
create role regression_user;
create role regresslo;
create role regress_rol_lock1;
create role regress_test_indirect;
create role regress_test_role;
create role regress_test_role2;
create role regress_test_role_super superuser;
create role regressuser1;
create role regressuser2;
create role regressuser3;
create role regressuser4;
create role regressuser5;
create role regtest_unpriv_user;
create role regtest_addr_user;
create role regtest_alter_user1;
create role regtest_alter_user2;
create role regtest_alter_user3;
create role rls_regress_group1;
create role rls_regress_group2;
create role rls_regress_user0;
create role rls_regress_user1;
create role rls_regress_user2;
create role rls_regress_exempt_user;
create role schemauser2;
create role seclabel_user1;
create role seclabel_user2;
create role selinto_user;
create role testrol1;
create role testrol2;
create role testrolx;
create role unprivileged_role;
create role "user";
create role view_user2;

\pset format unaligned
\pset tuples_only
\o ./sql/deparse_dump.sql

SELECT * FROM pg_deparse.output_commands();

\! @abs_builddir@/../../bin/psql/psql --dbname=@deparse_test_db@ -e < ./sql/deparse_dump.sql > results/deparse_dump.out 2>&1
