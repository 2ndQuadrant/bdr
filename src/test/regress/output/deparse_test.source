---
--- DEPARSE_TEST
---
\copy (SELECT command || ';' FROM deparse_test_commands ORDER BY id) TO './sql/deparse_dump.sql'
\! @psqldir@/psql --dbname=@deparse_test_db@ < ./sql/deparse_dump.sql > /dev/null
\! @psqldir@/pg_dump --schema-only --no-owner --no-privileges -Fp @deparse_test_db@ | perl -n -e 'print unless /^-- Dumped/'
--
-- PostgreSQL database dump
--


SET row_security = off;
SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

--
-- Name: plpgsql; Type: EXTENSION; Schema: -; Owner: -
--

CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;


--
-- Name: EXTENSION plpgsql; Type: COMMENT; Schema: -; Owner: -
--

COMMENT ON EXTENSION plpgsql IS 'PL/pgSQL procedural language';


SET search_path = public, pg_catalog;

--
-- Name: check_foreign_key(); Type: FUNCTION; Schema: public; Owner: -
--

CREATE FUNCTION check_foreign_key() RETURNS trigger
    LANGUAGE c
    AS '@libdir@/refint@DLSUFFIX@', 'check_foreign_key';


SET default_tablespace = '';

SET default_with_oids = false;

--
-- Name: hobbies_r; Type: TABLE; Schema: public; Owner: -; Tablespace: 
--

CREATE TABLE hobbies_r (
    name text,
    person text
);


--
-- PostgreSQL database dump complete
--

