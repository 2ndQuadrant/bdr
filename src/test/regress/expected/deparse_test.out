---
--- DEPARSE_TEST
---
\copy (SELECT command || ';' FROM deparse_test_commands ORDER BY id) TO './sql/deparse_dump.sql'
\! /space/sda1/ibarwick/2ndquadrant_bdr/src/test/regress/./tmp_check/install//home/ibarwick/devel/builds/DEPARSE/bin/psql --dbname=regression_deparse < ./sql/deparse_dump.sql > /dev/null
\! /space/sda1/ibarwick/2ndquadrant_bdr/src/test/regress/./tmp_check/install//home/ibarwick/devel/builds/DEPARSE/bin/pg_dump --schema-only --no-owner --no-privileges -Fp regression_deparse | egrep -v "^-- Dumped"
--
-- PostgreSQL database dump
--


SET row_security = off;
SET statement_timeout = 0;
SET lock_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SET check_function_bodies = false;
SET client_min_messages = warning;

--
-- Name: plpgsql; Type: EXTENSION; Schema: -; Owner: -
--

CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;


--
-- Name: EXTENSION plpgsql; Type: COMMENT; Schema: -; Owner: -
--

COMMENT ON EXTENSION plpgsql IS 'PL/pgSQL procedural language';


SET search_path = public, pg_catalog;

--
-- Name: check_foreign_key(); Type: FUNCTION; Schema: public; Owner: -
--

CREATE FUNCTION check_foreign_key() RETURNS trigger
    LANGUAGE c
    AS '/space/sda1/ibarwick/2ndquadrant_bdr/src/test/regress/refint.so', 'check_foreign_key';


SET default_tablespace = '';

SET default_with_oids = false;

--
-- Name: hobbies_r; Type: TABLE; Schema: public; Owner: -; Tablespace: 
--

CREATE TABLE hobbies_r (
    name text,
    person text
);


--
-- PostgreSQL database dump complete
--

